ui/src/pages/Lore/lore.scss

```diff
@@ -1,17 +1,76 @@
 .lore {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  &__content {
-    @extend .lore;
-    width: 80%;
+  width: 80%;
+  height: 70vh;
+  margin: 0 auto;
+  &__container {
+    overflow: hidden;
+    perspective: 240px;
+    perspective-origin: center 15%;
+    &__effect {
+      position: relative;
+      height: 64vh;
+      transform: rotateX(64deg) scale(0.55, 0.3) translateY(120vh);
+      animation: 40s linear 1s infinite translateText;
+      &__text {
+        color: maroon;
+        font-size: 5vh;
+        line-height: 1.5em;
+        font-weight: bold;
+        text-align: justify;
+      }
+    }
   }
 }
 
+#fade {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  height: 40vh;
+  background: linear-gradient(
+    0deg,
+    rgba(238, 203, 203, 0),
+    rgba(238, 203, 203, 0.25),
+    rgba(238, 203, 203, 1),
+    rgba(238, 203, 203, 1)
+  );
+  z-index: 10;
+}
+
+body {
+  overflow: hidden;
+}
```

Comment:
do we need to have body defined like this?

Reply:
I think this is a more efficient way of hiding overflow

Reply:
I deleted this and it seemed to work the same

Reply:
fuck out

Reply:
ok ok ill remove it lolol